backend/crawler/crawler.py

```diff
@@ -34,16 +81,71 @@ def run(crawler):
 
         driver.get('https://sugang.snu.ac.kr/sugang/cc/cc100.action')
         driver.implicitly_wait(1)
+        # detailed search
         driver.find_element_by_id('detail_button').click()
         driver.implicitly_wait(1)
+        # select year
         driver.find_element_by_xpath('//*[@id="srchOpenSchyy"]').clear()
         driver.implicitly_wait(1)
         driver.find_element_by_xpath('//*[@id="srchOpenSchyy"]').send_keys(crawler.year)
         driver.implicitly_wait(1)
+        # select semester
         driver.find_element_by_xpath('//*[@id="srchOpenShtm"]/option[{}]'.format(sid[crawler.semester])).click()
         driver.implicitly_wait(1)
+        # select bachelor's course
         driver.find_element_by_xpath('//*[@id="srchOpenSubmattCorsFg"]/option[2]').click()
         driver.implicitly_wait(1)
+
+        #driver.find_element_by_class_name('btn_search_ok').click()
+        #driver.implicitly_wait(1)
+
+        for type in tid:
+            crawl_type(crawler, driver, type)
+
+        #total_cnt = int(driver.find_element_by_xpath('//*[@id="content"]/div/div[3]/div[1]/div[1]/h3/span').text)
+        #total_page = ((total_cnt-1)//10)+1
+
+        #for i in range(1, total_page+1):
```

Comment:
Do you need these commented codes later?
If not, please remove unnecessary codes.